systemtest/060-delete.bats

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bats
+#
+# Copy tests
+#
+
+load helpers
+
+function setup() {
+    standard_setup
+
+    enable_delete=true
+    start_registry --enable-delete=$enable_delete reg
+}
+
+# delete image from registry
+@test "delete: remove image from registry" {
+    local remote_image=docker://busybox:latest
+    local localimg=docker://localhost:5000/busybox:unsigned
+    local output=
+
+    run_skopeo copy --dest-tls-verify=false $remote_image $localimg
+    output=$(run_skopeo inspect --tls-verify=false --raw $localimg)
+    echo $output | grep "vnd.docker.distribution.manifest.v2+json"
```

Comment:
What purpose does this inspect+grep serve?

Reply:
it's a known issue #667 , so I gave a extra check in here

Reply:
1. But #667 was (supposedly?) fixed in #680, so this check should not be necessary any more.
2. Even if we did need to have this safety check here, it would need an explicit comment explaining the purpose.